content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-v1.md

```diff
@@ -0,0 +1,7006 @@
+
+    1. 以特权身份运行
+    2. 具有 `CAP_SYS_ADMIN`
+
+    请注意，当 `spec.os.name` 为 "windows" 时，无法设置此字段。
+
+  <!--
+  - **securityContext.capabilities** (Capabilities)
+
+    The capabilities to add/drop when running containers. Defaults to the default set of capabilities granted by the container runtime. Note that this field cannot be set when spec.os.name is windows.
+
+    <a name="Capabilities"></a>
+
+    *Adds and removes POSIX capabilities from running containers.*
+
+    - **securityContext.capabilities.add** ([]string)
+
+      Added capabilities
+
+    - **securityContext.capabilities.drop** ([]string)
+
+      Removed capabilities
+  -->
+
+  - **securityContext.capabilities** (Capabilities)
+
+    运行容器时添加或放弃的权能（Capabilities）。默认为容器运行时所授予的权能集合。
+    注意，`spec.os.name` 为 "windows" 时不能设置此字段。
+
+    <a name="Capabilities"></a>
+    **在运行中的容器中添加和放弃 POSIX 权能。**
+
+    - **securityContext.capabilities.add** ([]string)
+
+      新增权能。
+
+    - **securityContext.capabilities.drop** ([]string)
+
+      放弃权能。
+
+  <!--
+  - **securityContext.seccompProfile** (SeccompProfile)
+
+    The seccomp options to use by this container. If seccomp options are provided at both the pod & container level, the container options override the pod options. Note that this field cannot be set when spec.os.name is windows.
+
+    <a name="SeccompProfile"></a>
+    *SeccompProfile defines a pod/container's seccomp profile settings. Only one profile source may be set.*
+  -->
+
+  - **securityContext.seccompProfile** (SeccompProfile)
+
+    此容器使用的 seccomp 选项。如果在 Pod 和容器级别都提供了 seccomp 选项，则容器级别的选项会覆盖 Pod 级别的选项设置。
+    注意，`spec.os.name` 为 "windows" 时不能设置此字段。
+
+    <a name="SeccompProfile"></a>
+    **SeccompProfile 定义 Pod/容器的 seccomp 配置文件设置。只能设置一个配置文件源。**
+
+    <!--
+    - **securityContext.seccompProfile.type** (string), required
+
+      type indicates which kind of seccomp profile will be applied. Valid options are:
+      
+      Localhost - a profile defined in a file on the node should be used. RuntimeDefault - the container runtime default profile should be used. Unconfined - no profile should be applied.
+    --> 
+
+    - **securityContext.seccompProfile.type** (string)，必需
+
+      type 指示应用哪种 seccomp 配置文件。有效的选项有：
+      
+      - `Localhost` - 应使用在节点上的文件中定义的配置文件。
+      - `RuntimeDefault` - 应使用容器运行时的默认配置文件。
+      - `Unconfined` - 不应用任何配置文件。
+     
+    <!--
+    - **securityContext.seccompProfile.localhostProfile** (string)
+
+      localhostProfile indicates a profile defined in a file on the node should be used. The profile must be preconfigured on the node to work. Must be a descending path, relative to the kubelet's configured seccomp profile location. Must only be set if type is "Localhost".
+-->
+    -->
+
+    - **securityContext.seccompProfile.localhostProfile** (string)
+
+      localhostProfile 指示应使用的在节点上的文件，文件中定义了配置文件。
+      该配置文件必须在节点上先行配置才能使用。
+      必须是相对于 kubelet 所配置的 seccomp 配置文件位置下的下级路径。
+      仅当 type 为 "Localhost" 时才必须设置。
```

Comment:
The comment is broken, translated text not shown.
Maybe we have to remove line 2101.